### DISCUSSION TITLE
AOSP 12.1 FLOSS vs. MinMicroG NoGoolag Edition

### DISCUSSION THREAD

**Discussion by GSIPRO** [phhusson/treble_experimentations#2311] [Q&A]
Is there a possibility to explicitly include the [MinMicroG NoGoolag Edition ↗](https://www.reddit.com/r/microg/wiki/minmicrog) in your builds of system-squeak AOSP 12.1?

**Comment by phhusson** [comment]
minimicrog is an installer, I can't embed an installer that doesn't make sense.
FLOSS variant already include most of the things listed.
Be explicit as to what change you want

**Reply by GSIPRO** [reply]
> FLOSS variant already include most of the things listed. Be explicit as to what change you want

»The NoGoolag edition is completley free of any Google or proprietary components other then Droidguard.« This is the key statement from [@FriendlyNeighborhoodShane ↗](https://github.com/friendlyneighborhoodshane/minmicrog_releases/releases) why I use MinMicroG NoGoolag Edition wherever possible.

**Reply by phhusson** [reply]
What do you think "FLOSS" variant mean?

**Reply by GSIPRO** [reply]
FLOSS (Free, Libre and Open Source Software) is descriptions for Free Software as well as FOSS (Free and Open Source Software).

microG is a free-as-in-freedom re-implementation of Google's proprietary Android user space apps and libraries. There are several pre-installed versions of microG, for example in /e/OS, CalyxOS, iodéOS and LineageOS-for-microG. microG can be brewed from different modules: GmsCore and UnifiedNlp, GsfProxy, etc.. With microG, the user can choose which services from Google they want to continue using, private data is obfuscated in any case. 

MinMicroG releases are available in two versions: The Standard Edition (ontains some proprietary Google Apps and libraries) and NoGoolag Edition (is completley free of any Google or proprietary components other then Droidguard.).

@phhusson treble experimentations [FLOSS versions include the following open source alternatives to GAPPS ↗](https://github.com/phhusson/treble_experimentations/wiki/Frequently-Asked-Questions-%28FAQ%29).  phh's Superuser, NewPipe, Silence, OsmAnd~, Lightning, Etar, Transportr, MuPDF viewer, AnySoftKeyboard, Yalp Store, K-9 Mail, Riot.im, DAVdroid, Nextcloud, F-Droid. `There is no mention of a microG edition.`

Since I assume against better knowledge that none of the above mentioned microG Edition is like MinMicroG NoGoolag Edition, i.e. completely free of any Google or proprietary components (except Droidguard), I took the liberty to ask you if the MinMicroG NoGoolag Edition are to be implemented in your GSI. 

If phhusson microG FLOSS GSI versions have the same features as @FriendlyNeighborhoodShane's MinMicroG NoGoolag Edition, then that would top it all off. If not, I will still continue to use the phhusson microG FLOSS GSI versions because phhusson GSI Updater ROMs are unmatched in quality.

**Reply by phhusson** [reply]
The source you're using is obsolete. The direct source is https://github.com/phhusson/vendor_foss/blob/master/update.sh#L99

it does include microg.

**Comment by GSIPRO** [comment]
https://github.com/phhusson/treble_experimentations/wiki refers to four headings: Home - GSI List - FAQ - Common Issues and Fixes.

FAQ (Frequently Asked Questions) uses an obsolete source, as I just learned from you. Okay!

MinMicroG NoGoolag Edition is a very good alternative. On ne peut rien y faire.

**Reply by phhusson** [reply]
Alternative to what? What is missing from my floss that you want to see in it?

**Reply by GSIPRO** [reply]
I'm looking at AOSP 12.1 FLOSS incl. microG for an explanation that is understandable to me as I find it at Official Minimal microG:  

AOSP 12.1 FLOSS edition is completley free of any Google or proprietary components other then Droidguard respectively AOSP 12.1 FLOSS edition edition contains some proprietary Google Apps and libraries.

**Reply by phhusson** [reply]
AOSP 12.1 FLOSS doesn't contain any proprietary component.

**Comment by GSIPRO** [comment]
`AOSP 12.1 FLOSS doesn't contain any proprietary component.`
Great! Thank you very much.
